common/buildcraft/factory/tile/ITileLed.java

```diff
@@ -0,0 +1,8 @@
+package buildcraft.factory.tile;
+
+public interface ITileLed {
+    boolean isDone();
```

Comment:
the led state is determined by if the machine is done or not but wouldn't isOn() or isLedOn() be more acurate?